cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f"{self.last_name} {self.first_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=50)
```

Comment:
The `name` field in the `Genre` model should have a `unique=True` constraint to ensure it matches the migration file and prevents duplicate genre names.